### PR TITLE
refactor Terraform to remove concurrency issue

### PR DESCRIPTION
**Pull request by Renetnielsen**
## Description

Before we were using the github actions matrix to handle multiple environments deployment.
We removed this to create dependencies between environments and to not have the concurrency issue there is when 4 deployments starts in parrallel

### CODE REVIEW

**Review by kristianschneider — APPROVED** [review]

File: .github/actions/terraform-cd/action.yml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+    ## CUSTOM DOMAIN STEPS
+    - name: Applying meteringpoint migrations
+      shell: bash
+      run: |
+        if ${{ inputs.ENVIRONMENT_NAME == 'Development' || inputs.ENVIRONMENT_NAME == 'Test' }}
+        then
+          echo "INCLUDE_TESTDATA=includeTestData" >> $GITHUB_ENV
+        fi
+        dotnet run --project source/Energinet.DataHub.MeteringPoints.ApplyDBMigrationsApp/ --configuration Release -- ${{ steps.terraform-apply.outputs.sql-server-connectionstring }} includeSeedData ${{ inputs.INCLUDE_TESTDATA }}
```
Comment (ZavezX) [inline .github/actions/terraform-cd/action.yml:151]: What is this and why is it in this domain?

File: .github/workflows/infrastructure-cd.yml [hunk 2]
```diff
@@ -27,111 +25,165 @@ env:
...
-          echo "ARM_SUBSCRIPTION_ID=${{ secrets.SUBSCRIPTION_ID }}" >> $GITHUB_ENV
-
       - name: Setup Terraform
         uses: hashicorp/setup-terraform@v1.2.1
         with:
           terraform_wrapper: false
           terraform_version: 0.14.10
-      # https://github.com/actions/setup-python>
-      - name: Setup Python
-        uses: actions/setup-python@v2
```
Comment (ZavezX) [inline .github/workflows/infrastructure-cd.yml:79]: Why was this removed?

File: build/infrastructure/variables.tf [hunk 3]
```diff
@@ -66,11 +65,6 @@ variable "grid_loss_sys_cor_path" {
   description   = "Path to location of system correction and grid loss used in aggregation job"
 }
 
-variable "database_password" {
-    type        = string
-    description = "meta database password"
-}
```
Comment (ZavezX) [inline build/infrastructure/variables.tf:72]: Why was this removed? Do we access this via secrets in github instead?

**Comment by Renetnielsen** [comment]
@ZavezX I will move this to draft, since it was not ready yet. My bad :)

File: .github/workflows/infrastructure-cd.yml (hunk 2 above)
Comment (Renetnielsen) [reply .github/workflows/infrastructure-cd.yml:79]: It is not used anywhere in TF.

File: build/infrastructure/variables.tf (hunk 3 above)
Comment (Renetnielsen) [reply build/infrastructure/variables.tf:72]: It is not used anywhere in TF.

**Comment by Renetnielsen** [comment]
@kristianschneider @ZavezX Please re-review

**Review by mknic — APPROVED** [review]

**Review by ZavezX — APPROVED** [review]

**Review by kristianschneider — APPROVED** [review]
